Source/NSMenu.m

```diff
@@ -280,6 +282,12 @@ - (void) _organizeMenu
       appItem = [self itemWithTitle: appTitle];
       appMenu = [appItem submenu];
 
+      if ([[GSTheme theme] organizesMainMenu])
```

Comment:
This can get rather confusing. now the theme may rearrange the menu and afterwords the code here will do something different with it. Wouldn't it be simpler to move all the code from here into the default implementation in the theme and allow other themes to override it?

Reply:
You know, I was thinking about that -- there isn't any really good reason for that not to be the case. The whole point of GSTheme, as I understand it, is to allow overriding default behaviour without breaking existing code -- so by moving the default behaviour of things into the base GSTheme class, we would accomplish that end with more flexibility for theme authors and no cost to those who aren't interested in fancy new themes.

So anyway, I'll get that change done too.